server/src/main/java/org/opensearch/tasks/CancellableTask.java

```diff
@@ -98,22 +103,18 @@ public boolean cancelOnParentLeaving() {
     }
 
     public Long getCancellationStartTime() {
-        return cancellationStartTime;
+        return Objects.requireNonNull(cancelledInfo.get()).cancellationStartTime;
```

Comment:
Previously, this method returned `null` when the cancellation hasn't happened. But now it'll throw a NullPointerException. Can we preserve the original behaviour?

Reply:
Oh shoot good call. I will swap it back.